### DISCUSSION TITLE
Blue- versus red-colored box plots in subsampling-based VIMP inference for survival analysis

### DISCUSSION THREAD

**Discussion by rachaelvp** [kogalur/randomForestSRC#437] [Q&A]
Hi! I am following your [vignette on random survival forests](https://www.randomforestsrc.org/articles/survival.html)—very helpful, thank you—and I am confused about the colors on the variable importance plot. Specifically, when we use subsampling to estimate inference some of the features appear as blue box plots while others are red. Could you please explain what this color difference means or refer me to the answer if you've already addressed this elsewhere?

**Comment by ishwaran** [comment]
Blue are variables with confidence intervals including 0 - therefore they are not statistical significant.

Red are variables where the opposite happens (CI excludes 0 - therefore they are significant).
